runtime/src/mm/cpp/GlobalData.hpp

```diff
@@ -29,8 +29,9 @@ class GlobalData : private Pinned {
 
 private:
     GlobalData();
-    ~GlobalData();
+    ~GlobalData() = delete;
 
+    // This `GlobalData` is never destroyed.
```

Comment:
With the new MM we don't shutdown the runtime at the program finish, so no need to destroy `GlobalData` too.